ccronexpr/LICENSE.txt

```diff
@@ -0,0 +1 @@
+ccronexpr/LICENSE.txt
```

Comment:
Symlinks are not supported here, I'm afraid. Please copy the license file from the original project.